### PR TITLE
Port To Python 3

### PR DESCRIPTION
**Pull request by Creatune**
Worked fine before and after porting. I also updated the sugargame repo and tested again. It seemed to work well. Please let me know if there are any more changes that I can make.

### CODE REVIEW

**Comment by quozl** [comment]
Thanks.  Please complete the xxx_todo_changeme sections; these are not intended to be left as-is.

**Comment by Creatune** [comment]
From this link:
https://github.com/mitsuhiko/python-modernize/issues/15
I have inferred that instead of the numerous xxx_todo_changeme sections in the edited files, I should just change the name according to the original variables used. So should I do just that, or is there anything else that I can do regarding this?

**Comment by Creatune** [comment]
I changed all the xxx_todo_changeme to var(number) to maintain a uniform flow. If there are any issues with the names, I will gladly make further changes.

**Comment by quozl** [comment]
Thanks, but that's just as bad as leaving it unchanged.  Please give each variable a meaningful name that describes either what it contains, what it is for, how it is used, or some other semantic.  Use existing naming convention if possible.  See the other variable names in the argument list for the methods.

**Comment by Creatune** [comment]
I thought so too. I will be on with the task right now :)

**Comment by Creatune** [comment]
My apologies for the multiple commits. I didn't notice a few required changes.

**Comment by quozl** [comment]
Thanks.  I'll look at this tomorrow, my time.

**Comment by quozl** [comment]
You seem to be struggling to choose symbolic names; I guess you haven't done it much.  I'm torn between finishing it for you or trying to explain further.  I'll try to explain further;

* the name need only be unique within the scope of the function; you seem to have tried to use a different name for each function,
* the name should be as short as possible but still convey enough about the value that someone reading the code will be able to understand it, for as long as they need to; which is two lines,

For a good reference on choosing names, see Thinking Forth, by Leo Brodie.  While it has a focus on the Forth language, much of what it says about problem solving applies across languages.

Others in Sugar Labs have hit the same problem with porting code by the same author;
* in the Triples activity, the same code was changed to _coordinates_ and _color_, https://github.com/sugarlabs/triples/commit/b6c29dfdb7934d343a0f88d0a43f26132634b322 by @pro-panda,
* in the Across and Down activity, https://github.com/sugarlabs/across-and-down-activity/commit/b2a3d1e773bff938bad1aa812baed9c0d80534b2 to _coordinates_ and _color_, by @Aniket21mathur,
* in the Countries activity, https://github.com/sugarlabs/countries-activity/commit/d72ba227c2e5f25a0789b80756ba2bf7e7719485 wasn't changed, and was left as _xxx_todo_changeme_, by @chimosky.
* in the Letters activity, my commit https://github.com/sugarlabs/letters/commit/062795a8563039b6892bad940d883e85e3f22a4f used _here_ for a list of coordinates, and _rgb_ for a list of colour values; but my change wasn't reviewed, so I'm not saying it is the best way,

It is a mess.  :grin:

**Comment by Creatune** [comment]
I am so sorry for the delay. I had completely forgotten about this PR. Anyways, Merry Christmas!! :)

File: utils.py [hunk 1]
```diff
@@ -78,13 +78,18 @@ def shuffle(lst):
         lt.append(lst[r]); l1.remove(lst[r])
     return lt
 
-def centre_blit(screen,img,(cx,cy),angle=0): # rotation is clockwise
+#The word 'cord' used later on in various variables refers to the word 'co-ordinates'
+
```
Comment (quozl) [inline utils.py]: Above two changed lines are no longer required.

File: buttons.py [hunk 2]
```diff
@@ -4,7 +4,8 @@
 class Button:
     _instances=[]
     
-    def __init__(self,name,(x1,y1),centre=True): # eg ('plus',(30,40))
+    def __init__(self,name, coordinates,centre=True): # eg ('plus',(30,40))
```
Comment (quozl) [inline buttons.py]: Usually we would have spaces after commas in an argument list, but on the other hand if that would require a huge change for consistency, then best not to.

File: utils.py [hunk 3]
```diff
@@ -78,13 +78,18 @@ def shuffle(lst):
...
+def centre_blit(screen,img, coordinates,angle=0): # rotation is clockwise
+    (cx,cy) = coordinates
     img1=img
     if angle!=0: img1=pygame.transform.rotate(img,-angle)
     rect=img1.get_rect()
     screen.blit(img1,(cx-rect.width/2,cy-rect.height/2))
 
-def text_blit(screen,s,font,(cx,cy),(r,g,b),shadow=True):
+def text_blit(screen,s,font, coordinates, color,shadow=True):
```
Comment (quozl) [inline utils.py]: Mix of using spaces or not using spaces; please use existing style in the file, unless all such lines are changed in which case uses a space after comma.

**Comment by quozl** [comment]
Thanks.  Reviewed.  Review added as commits.

**Comment by Creatune** [comment]
Thanks! I will keep these details in mind next time.

**Pull request merged**